src/compute/aggregate/simd/packed.rs

```diff
@@ -33,22 +33,22 @@ macro_rules! simd_ord_int {
 
             #[inline]
             fn max_element(self) -> $type {
-                self.max_element()
+                self.horizontal_max()
             }
 
             #[inline]
             fn min_element(self) -> $type {
-                self.min_element()
+                self.horizontal_min()
             }
 
             #[inline]
             fn max(self, x: Self) -> Self {
-                self.max(x)
+                std::cmp::Ord::max(self, x)
```

Comment:
if you want element-wise min/max, this'll give the wrong answer

Reply:
thanks! Fixed.